.gitignore

```diff
@@ -1,5 +1,6 @@
 /.bundle/
 /.yardoc
+/.idea/
```

Comment:
Please don't add non-project related files to project gitignore. Set up and use global gitignore file on your machine instead. See https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer
```suggestion
```

Reply:
Ok, I'll fix it